CHANGES.md

```diff
@@ -56,13 +56,44 @@
   
   the load order is:
 
-  - ./config/twitcher
   - ./config/magpie (loaded as a dependency of twitcher, not loaded a second time after project-api)
-  - ./config/project-api 
-  - ./optional-components/generic_bird
+  - ./config/twitcher
+  - ./config/project-api
   - ./config/wps_outputs-volume (loaded as a dependency of generic_bird)
+  - ./optional-components/generic_bird
   - ./components/cowbird
 
+  This load order also applies to the order that docker-compose-extra.yml files are specified. If a component also
+  includes an override file for another component (eg: ./config/finch/config/proxy/docker-compose-extra.yml overrides 
+  ./config/proxy/docker-compose-extra.yml), the following additional load order rules apply:
+
+  - if the component that is being overridden has already been loaded, the override file is loaded immediately
+  - otherwise, the override files will be loaded immediately after the component that is being overridden has been loaded
```

Comment:
Very nice delayed load !  Components do not have to know the load ordering amount themselves.